README.md

```diff
@@ -245,10 +245,11 @@ TSLint ships with a set of core rules that can be configured. However, users are
 
 Rule names are always camel-cased and *must* contain the suffix `Rule`. Let us take the example of how to write a new rule to forbid all import statements (you know, *for science*). Let us name the rule file `noImportsRule.ts`. Rules can be referenced in `tslint.json` in their dasherized forms, so `"no-imports": true` would turn on the rule.
 
-Now, let us first write the rule in TypeScript. At the top, we reference TSLint's [definition](https://github.com/palantir/tslint/blob/master/lib/tslint.d.ts) file. The exported class name must always be named `Rule` and extend from `Lint.Rules.AbstractRule`.
+Now, let us first write the rule in TypeScript. At the top, we reference TSLint's [definition](https://github.com/palantir/tslint/blob/master/lib/tslint.d.ts) file and the [definition file](https://github.com/palantir/tslint/blob/master/typings/typescriptServices.d.ts) for TypeScript's services. The exported class name must always be named `Rule` and extend from `Lint.Rules.AbstractRule`.
```

Comment:
some nitpicks:

- make the first link (to tslint.d.ts) also apply to the word "file" (so both links are "definition file")
- TypeScript's services -> TypeScript's language services
- "class name must always be named" -> "class must always be named"